flask/app.py

```diff
@@ -994,22 +994,41 @@ def make_null_session(self):
         return self.session_interface.make_null_session(self)
 
     @setupmethod
-    def register_blueprint(self, blueprint, **options):
-        """Registers a blueprint on the application.
+    def register_blueprint(
+        self, blueprint, **options
```

Comment:
Oops, that's leftover from me thinking about adding the params to the signature.